index.d.ts

```diff
@@ -524,15 +524,20 @@ declare namespace Moleculer {
 		[name: string]: any;
 	}
 
-	type ServiceEventHandler = ((payload: any, sender: string, eventName: string) => void) & ThisType<Service>;
+	type ServiceEventOldHandler = ((payload: any, sender: string, eventName: string, ctx: Context) => void) & ThisType<Service>;
+
+	type ServiceEventHandler = ((ctx: Context) => void) & ThisType<Service>;
```

Comment:
It should be noted in the PR that this is a _breaking_ change.  Since 0.14 is currently in beta, that should be fine and it is a good time to get the breaking change in place.

Reply:
Renamed as suggested and added note about being a breaking change. Thank you